src/main/java/com/ratifire/devrate/service/interview/InterviewCompletionService.java

```diff
@@ -60,10 +67,42 @@ public String completeInterviewProcess(Meeting meeting) {
     Long interviewerFeedbackDetailId = interviewFeedbackDetailId.get("interviewerFeedbackId");
     notificationService.addInterviewFeedbackDetail(interviewer, interviewerFeedbackDetailId);
 
+    try {
+      zoomApiService.deleteMeeting(Long.parseLong(meeting.getId()));
+    } catch (ZoomApiException e) {
+      logger.error("Zoom API exception occurred while trying to delete the meeting "
+          + "with meetingId: {}. {}", meeting.getId(), e.getMessage());
+    }
     interviewService.deleteInterview(interview.getId());
     interviewRequestService.deleteInterviewRequests(
         List.of(interview.getInterviewerRequest().getId(),
             interview.getCandidateRequest().getId()));
     return "Interview process completed successfully!";
   }
+
+  /**
+   * Validates if the webhook is triggered at the right time.
+   *
+   * @param payload The event data containing information about the meeting.
+   * @return boolean indicating if the webhook can proceed with interview completion.
+   * @throws ZoomWebhookException If the data is invalid.
+   */
+  public boolean validateMeetingEndTime(WebHookRequest payload) throws ZoomWebhookException {
```

Comment:
In my opinion, it's better to move this method to another validation class. Moving it to a dedicated class improves modularity, makes the code more maintainable. But it's up to you